Dockerfile-dev

```diff
@@ -16,6 +16,13 @@ WORKDIR /app
 
 RUN env | sort
 
+# Ensure changes to build doesn't re-install requirements
+WORKDIR /tmp/app
+COPY bpa-ingest /tmp/app/bpa-ingest
+COPY bpaworkflow/pyproject.toml bpaworkflow/poetry.lock /tmp/app/bpaworkflow/
+RUN cd /tmp/app/bpaworkflow && poetry install --no-root
+WORKDIR /app
+RUN rm -rf /tmp/app
 # Copy code and install the app
 COPY . /app
 RUN cd /app/bpaworkflow && poetry install
```

Comment:
I don't understand what this change achieves – we still run the poetry install in /app, what does doing all this stuff in /tmp (which we then seem to delete) achieve?

Reply:
this was just poc code in dev for keeping versions from reinstalling -deleted now